terraform/environments/core-network-services/vpc.tf

```diff
@@ -33,4 +33,25 @@ module "vpc_hub" {
   # Tags
   tags_common = local.tags
   tags_prefix = each.key
+}
+
+resource "aws_route" "modernisation_platform_10-20-0-0--16" {
+  for_each               = local.networking
+  destination_cidr_block = "10.20.0.0/16"
+  gateway_id             = aws_ec2_transit_gateway.transit-gateway.id
+  route_table_id         = module.vpc_hub[each.key].public_route_tables.id
+}
+
+resource "aws_route" "modernisation_platform_10-25-0-0--16" {
```

Comment:
What is this range?  I see the core account range 10.20 above, and 10.26 is dev and test but I'm not familiar with this one should it be 10.27?  Is it worth adding a comment so we know what each of the ranges are?

Reply:
You're totally right! I've pushed a fix for this. Good catch!